examples/motion_viewer.py

```diff
@@ -97,14 +127,25 @@ def key_press_event(self, event: Application.KeyEvent) -> None:
                 logger.info("Command: hide model")
             else:
                 logger.info("Command: load model")
+                self.remove_short_path_traj_obj()
                 self.fm_demo.load_model()
 
         elif key == pressed.J:
-            if not self.sim.pathfinder.is_loaded:
+            if event.modifiers == mod.ALT:
+                self.simulating = False
+                self.fm_demo.update_pathfollower_sequential(
+                    path_time=random.uniform(0.0, 10.0)
+                )
```

Comment:
Can this be called before `setup_pathfollower`? If not you should add some protection/warning from potential crash.

Reply:
I do with this line
```
# precondition
        if not all(
            [self.model, self.path_points, self.activity == Activity.PATH_FOLLOW_SEQ]
        ):
```